### ISSUE TITLE
Stable v4 released

### ISSUE THREAD

**Issue by wBureshGitHub** [degouville/atom-bootstrap4#22]
Bootstrap 4 has finally arrived! How about updating your very useful package?

**Comment by hieuhuynh93** [comment]
Yeah, I'm also looking forward to it. Hope he will update it soon

**Comment by degouville** [comment]
Hi @wBureshGitHub, @hieuhuynh93, new updates are planned for next month.
However you can already help by sending me pull requests w/ your favorites feature/snippets.

Peace ✌️.

**Comment by hieuhuynh93** [comment]
Hi @degouville , thanks for your good news. Now I'm so satisfied with it. But i will submit the pull requests if I figure out any ideas.

**Comment by hieuhuynh93** [comment]
Hi, I have a question. Why this package not same your package on Github ? https://atom.io/packages/atom-bootstrap4

Or you still not release latest version to atom.io ?
